website/docs/r/policy_tier0_gateway_gre_tunnel.html.markdown

```diff
@@ -0,0 +1,130 @@
+---
+subcategory: "Gateways and Routing"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_tier0_gateway_gre_tunnel"
+description: A resource to configure a GRE tunnel on Tier-0 gateway on NSX Policy manager.
+---
+
+# nsxt_policy_tier0_gateway_gre_tunnel
+
+This resource provides a method for the management of a Tier-0 gateway GRE tunnel. Note that edge cluster and in interface must be configured on Tier-0 Gateway in order to configure GRE tunnels on it.
```

Comment:
User-level comment: What's the "in" interface? It's not a term I'm familiar with for Tier-0 gateways

Reply:
Should be "an interface" :)